scripts/apps/users/directives/UserPreferencesDirective.ts

```diff
@@ -16,6 +16,68 @@ import {applyDefault} from 'core/helpers/typescript-helpers';
 UserPreferencesDirective.$inject = ['session', 'preferencesService', 'notify', 'asset',
     'metadata', 'desks', 'modal', '$timeout', '$q', 'userList', 'lodash', 'search'];
 
+function translateCategoryName(category) {
+    switch (category.name.toLowerCase()) {
+    case 'advisory': return gettext('Advisory');
+    case 'agate': return gettext('Agate');
+    case 'atlantic': return gettext('Atlantic');
+    case 'audio skeds': return gettext('Audio Skeds');
+    case 'business': return gettext('Business');
+    case 'entertainment': return gettext('Entertainment');
+    case 'finance': return gettext('Finance');
+    case 'international': return gettext('International');
+    case 'lifestyle': return gettext('Lifestyle');
+    case 'national': return gettext('National');
+    case 'ontario/quebec': return gettext('Ontario/Quebec');
+    case 'prairies/bc': return gettext('Prairies/BC');
+    case 'press release': return gettext('Press Release');
+    case 'spare news': return gettext('Spare News');
+    case 'spare sports': return gettext('Spare Sports');
+    case 'sports': return gettext('Sports');
+    case 'travel': return gettext('Travel');
+    }
+
+    return category.name;
+}
+
+function translatePreferenceLabel(label) {
+    switch (label.toLowerCase()) {
+    case 'allow desktop notifications': return gettext('Allow Desktop Notifications');
```

Comment:
Since the list of preferences is coming from the back-end with labels already present, it should come already translated. Create a task for someone to fix this on the back-end.